jsm.go

```diff
@@ -170,6 +170,8 @@ type accountInfoResponse struct {
 }
 
 // AccountInfo retrieves info about the JetStream usage from the current account.
+// If JetStream is not enabled, this will return ErrJetStreamNotEnabled
```

Comment:
This now has to be the case for every API call to the JS context, but I think that is ok.

Reply:
@wallyqs we need to probably go through our public calls and make sure that is so

Reply:
yes could happen if JS becomes disabled